source/ConfigSetup.h

```diff
@@ -82,5 +82,6 @@ EMP_BUILD_CONFIG(SymConfigBase,
     VALUE(THREAD_COUNT, size_t, 12, "Number of threads used to process organisms in parallel"),
     VALUE(RANDOM_ANCESTOR, bool, false, "Randomize ancestor genomes instead of using the blank genome with just NOT and reproduction"),
     VALUE(TASK_TYPE, bool, 0, "0 for squaring tasks, 1 for logic tasks"),
+    VALUE(DONATION, bool, 0, "1 if you want organisms to have a donate instruction, 0 if not"),
```

Comment:
Let's make this more detailed to be clear, "DONATE_INST" and then "if you want DONATE instruction in the instruction set"